app/_src/mesh/features/windows.md

```diff
@@ -2,7 +2,7 @@
 title: Kong Mesh with Windows
 ---
 
-{% if_version gte:2.4.x %}
+{% if_version gte:2.4.x lt:2.11.x %}
```

Comment:
```suggestion
{% if_version gte:2.4.x lte:2.11.x %}
```

Reply:
Is `lt` not supported? I only want less than 2.11, I guess `lte:2.10.x` then?

Reply:
oh that is supported. normally we do `gte` and `lte` for easier search across the platform (for consistency).

Though - looks like this page was removed from the latest versions some time back, so it doesn't actually matter. 
In fact, here's the page in 2.0: https://deploy-preview-8100--kongdocs.netlify.app/mesh/2.0.x/features/windows/ - totally blank. And it doesn't exist from 2.4 onward.

Reply:
Ye, I'm going to put it back in between 2.4 and 2.11